ArtOfIllusion/src/artofillusion/procedural/Module.java

```diff
@@ -285,11 +286,15 @@ public boolean edit(BFrame fr, Scene theScene)
     return false;
   }
 
-  /** This method initializes the module in preparation for evaluating the procedure at a
-      new point.  The default implementation does nothing.  Subclasses whose output depends
-      on the point should override this method. */
+  /**
+   * This method initializes the module in preparation for evaluating the
+   * procedure at a new point.The default implementation does nothing.
+   * Subclasses whose output depends on the point should override this method.
+   *
+   * @param point
```

Comment:
@param annotations in the javadoc should add information beyond what can be discerned from the type signature. Non-obvious use details, etc.